test/Observable_mockito_test.dart

```diff
@@ -0,0 +1,49 @@
+library wCommon.test;
+
+// This is a working example of Dart unit tests using the mockito mocking library and unittest for
+// assertions.  It tests the same functionality as Observable_stock_test.dart.
+
+// Testing imports
+import 'package:unittest/unittest.dart';
+import 'package:mockito/mockito.dart';
+import 'package:unittest/html_enhanced_config.dart';
+
+// Testing dependencies
+import 'dart:async';
+import 'dart:js';
+import 'package:wCommon/wCommon.dart';
+
+class MockJsObject extends Mock {}
+
+main() {
+
+  useHtmlEnhancedConfiguration();
```

Comment:
useHtmlEnhancedConfiguration?  It configures the test runner output when running tests in Dartium.  It's probably all going away with test 0.12.